packages/next/src/client/components/reducer.ts

```diff
@@ -914,7 +917,8 @@ function clientReducer(
           href
         )
 
-        // Copy subTreeData for the root node of the cache.
+        // Copy subTreeData for the root node of the cache.              cache.status = CacheStates.READY
```

Comment:
nit: think you got an extra bit of comment here

Reply:
Strange! Fixed <picture data-single-emoji=":lolsob:" title=":lolsob:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..3c6F-ebvnUPtoPHJ.PjiYQOvpaHE1Tp79_L-LbKZPegaCYH8Vq7zzhxs6SifT9IVbPjslnYH-M0BruQSTEdxcr_5SmzYyVH51PAET0NbvKtagCvcKo3tG19PKoKNThxnytUjh.BOSN8-Fw2FUSCkfgOjTqCw" alt=":lolsob:" align="absmiddle"></picture>